tools/ArduinoCsCompiler/Frontend/Frontend.csproj

```diff
@@ -9,20 +9,44 @@
     <AssemblyName>acs</AssemblyName>
```

Comment:
I just noticed this project is targeting net6.0, why is that? Should it be net8.0 instead? `net6.0` is out of support so we'd want to update that now.

Reply:
The entire library still targets 6.0, so this is consistent. But more importantly is the fact that the runtime used affects the compilation. Because acs relies heavily on the internal runtime implementations (eg the native methods called) updating this requires considerable effort and testing. 

So this is something for a future update (I'm looking into a slightly different approach there, though)